src/commands/games/minecraft/modpack.js

```diff
@@ -0,0 +1,36 @@
+const { SearchCommand, SwitchbladeEmbed, Constants } = require('../../../')
+const moment = require('moment')
+
+module.exports = class MinecraftModpack extends SearchCommand {
+  constructor (client) {
+    super({
+      name: 'modpack',
+      aliases: ['curseforge', 'm', 'twitch'],
+      parent: 'minecraft',
+      embedColor: Constants.CURSEFORGE_COLOR,
+      embedLogoURL: 'https://minecraft-hosting.com/wp-content/uploads/2019/01/curseforge.png'
```

Comment:
Please refrain from using third party websites. At best, [upload the image to Imgur](https://imgur.com)

Reply:
I mean, imgur is a third party website